Angular/Angular14/original/angular.json

```diff
@@ -99,19 +105,6 @@
             "scripts": []
           }
         },
-        "lint": {
```

Comment:
is lint disabled intentionally, or by occasion, during local debug?

Reply:
It was removed by the Angular built-in updater - I didn't remove it manually. So I thought that it should be kept as-is

Reply:
ok